pom.xml

```diff
@@ -287,6 +287,7 @@
                         <configuration>
                             <excludes>
                                 <exclude>checkstyle:/src/test/java/com/jcabi/github/NullabilityTest.java</exclude>
+                                <exclude>checkstyle:/src/main/java/com/jcabi/github/RtDatatypeConverter.java</exclude>
```

Comment:
@pecko let's add a puzzle to re-design this class and remove it from exclusions.